.nuspec/Microsoft.Maui.Core.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+
+  <PropertyGroup Condition=" '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'windows' ">
```

Comment:
Moved the properties into this props file so that it can be shared by the workloads and the repo.